sympy/physics/mechanics/joint.py

```diff
@@ -556,7 +556,8 @@ def create_symbol(number):
 class PinJoint(Joint):
     """Pin (Revolute) Joint.
 
-    .. image:: PinJoint.svg
+    .. raw:: html
+        :file: ../../../doc/src/modules/physics/mechanics/api/PinJoint.svg
```

Comment:
Changed it for the `PinJoint`. Don't really like specifying the path here, but it does fix it.